sdg/outputs/OutputGeoJson.py

```diff
@@ -0,0 +1,263 @@
+                    json.dump(geometry_data, f)
+
+        return status
+
+
+    def clean_disaggregations(self, disaggregations):
+        """Apply any modifications to disaggregations before saving them into
+        the GeoJSON file.
+
+        Parameters
+        ----------
+        disaggregations : dict
+            A dict of disaggregations, with category keyed to subcategory.
+
+        Returns
+        -------
+        dict
+            A modified version of the disaggregations dict.
+        """
+        # We don't need the actual geocode column.
+        del disaggregations[self.id_column]
+        # Remove any others necessary.
+        for column in self.exclude_columns:
+            if column in disaggregations:
+                del disaggregations[column]
+        # Convert null/nan/etc into just null.
+        for key in disaggregations:
+            if pd.isna(disaggregations[key]):
+                disaggregations[key] = None
+        return disaggregations
+
+
+    def replace_geocode(self, geocode):
+        """Make any replaces of geocodes, according to the id_replacements.
+
+        Parameters
+        ----------
+        geocode : string
+            A geocode to look for a replacement for.
+
+        Returns
+        -------
+        string
+            The replaced geocode, or the original, if no replacement was found.
+        """
+        if geocode in self.id_replacements:
+            return self.id_replacements[geocode]
+        return geocode
+
+
+    def validate(self):
+        """Validate geojson-related source data. Overrides parent.
+
+        This output does not currently test general data and metadata, as it is
+        assumed that this is a supplemental output, and that another output will
+        validate all of that.
+        """
+
+        status = True
+
+        # Make sure the geojson_file has the required properties on each feature.
+        for index, feature in enumerate(self.geometry_data['features']):
+            feature_is_bad = False
+            if self.name_property not in feature['properties']:
+                print('Name property "' + self.name_property + '" not found in feature:')
+                feature_is_bad = True
+            if self.id_property not in feature['properties']:
+                print('ID property "' + self.id_property + '" not found in feature:')
+                feature_is_bad = True
+            if feature_is_bad:
+                print(feature['properties'])
+                status = False
+
+        # Make sure at least one indicator has geocodes.
+        no_indicators_have_geocodes = True
+        for inid in self.indicators:
+            if self.indicator_has_geocodes(self.indicators[inid]):
+                no_indicators_have_geocodes = False
+                break
+
+        if no_indicators_have_geocodes:
+            print('No indicators have data under "' + self.id_column + '".')
+            status = False
+
+        return status
```

Comment:
This could be separated in its own function `geocodes_exist` (or something similar)

Reply:
Agreed, I made that change.